src/subscription/subscriptions-reducer.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { createAction } from '@bigcommerce/data-store';
+
+import { RequestError } from '../common/error/errors';
+import { getErrorResponse } from '../common/http-request/responses.mock';
+import { SubscriptionsActionType } from '../subscription';
+
+import subscriptionsReducer from './subscriptions-reducer';
+import SubscriptionsState from './subscriptions-state';
+
+describe('billingAddressReducer', () => {
```

Comment:
This is probably a copy-and-paste mistake. Can you please rename this to `subscriptionsReducer` instead?